src/accelerator.rs

```diff
@@ -98,6 +98,16 @@ impl Accelerator {
         self.id
     }
 
+    /// Returns the modifier for this accelerator
+    pub fn modifiers(&self) -> &Modifiers {
+        &self.mods
+    }
+
+    /// Returns the code for this accelerator
+    pub fn key(&self) -> &Code {
+        &self.key
+    }
```

Comment:
`Modifiers` and `Code` are copy types so we can just return them without a reference.

```suggestion
    /// Returns the modifier for this accelerator
    pub fn modifiers(&self) -> Modifiers {
        self.mods
    }

    /// Returns the code for this accelerator
    pub fn key(&self) -> Code {
        self.key
    }
```